index.html

```diff
@@ -405,39 +515,75 @@ <h3 id="placement-of-mono-ruby">Placement of mono-ruby</h3>
 <section>
 <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
 
-<p>Group-ruby is placed as follows:</p>
+<p>
+<span class="checkme">
+In this section, placement rules of group-ruby are described as combinations 
+of two groups of characters, 
+one as 'Western characters' which has proportional width 
+and consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-24">Grouped numerals (cl-24)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-25">Unit symbols (cl-25)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Western word space (cl-26)</a>,
+and 
+<a href="https://www.w3.org/TR/jlreq/#cl-27">Western characters (cl-27)</a>
+[[JLREQ]], 
+and another as 'Kana characters' which consisted with characters like 
+<a href="https://www.w3.org/TR/jlreq/#cl-15">Hiragana (cl-15)</a>,
+<a href="https://www.w3.org/TR/jlreq/#cl-26">Katakana (cl-16)</a>, and 
+<a href="https://www.w3.org/TR/jlreq/#cl-19">Ideographic characters (cl-19)</a>
+[[JLREQ]]. 
+For Western characters, strings are read by clusters of multiple characters, 
+adding spacing between characters for justification are wanted to avoid. 
+</span>
+The way they are positioned depends
+on how their respective lengths would
+compare if they were each laid out
+without any inter-letter spacing.
+When their respective lengths would be the same,
+both are laid out without inter-letter spacing
+and placed such that their respective centers in the inline direction are aligned
+(see <a href="#group-ruby"></a>).
+<span class="checkme">
+For other cases, the placement depends on the following:
+</span>
+</p>
+<figure id=group-ruby>
+  <img src="img/fig12.svg" />
+  <figcaption>Example 1 of group-ruby</figcaption>
+</figure>
+
+<aside class=note title="Inter-letter spacing in group-ruby">
+<span class="checkme">
+In the case of group-ruby, it is desired to align the length of ruby string 
+and the length of ruby base character string, but these lengths could differ 
+by numbers of characters. To compensate their difference, adding spacing 
+to preceding, following, and inter-letter of ruby string or ruby base 
+character string is used for Kana characters. 
+Adding spacing by a ratio of 1 to preceding and following, and 2 to 
+inter-letter was noted as a method to make well balanced layout in letterpress 
+printing, and is used in following method. 
+In contrast, for Western characters, this note focuses on keeping clusters of 
+Western characters rather than alignment of lengths, and does not take 
+a method to have inter-letter spacing. 
+</span>
+</aside>
+
+<p>
+<span class="checkme">
+To align following items to the two step processing method described in 
```

Comment:
I think we can add an id to the `li` element of the two-step processing method and link to it directly.

Reply:
to be done as a part of #22